### PR TITLE
Dataview.prototype.get* methods

### PR DESCRIPTION
**Pull request by leobalter**
The feedback from #630 was also applied on this branch.

This and #630 covers the get* methods.

### CODE REVIEW

File: test/built-ins/DataView/prototype/getFloat64/return-nan.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+sample.setUint8(0, 127);
+sample.setUint8(1, 251);
+sample.setUint8(2, 0);
+sample.setUint8(3, 0);
+sample.setUint8(4, 0);
+sample.setUint8(5, 0);
+sample.setUint8(6, 0);
+sample.setUint8(7, 0);
+assert.sameValue(sample.getFloat64(0), NaN, "127, 251, 0, ...");
```
Comment (jugglinmike) [inline test/built-ins/DataView/prototype/getFloat64/return-nan.js]: Very nice

File: test/built-ins/DataView/prototype/getFloat64/minus-zero.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+sample.setUint8(2, 0);
+sample.setUint8(3, 0);
+sample.setUint8(4, 0);
+sample.setUint8(5, 0);
+sample.setUint8(6, 0);
+sample.setUint8(7, 0);
+
+var result = sample.getFloat64(0);
+assert.sameValue(result, 0);
+assert.sameValue(1 / result, -Infinity);
```
Comment (jugglinmike) [inline test/built-ins/DataView/prototype/getFloat64/minus-zero.js]: Just like the operation in the spec (and as you realized in gh-630),
SameValue accounts for negative zero. As written, the first assertion is
incorrect; we'll need to compare `result` to `-0` instead.

**Comment by jugglinmike** [comment]
For the following pairs of files, the chosen values  do not demonstrate the distinction between Int16 and Uint16:

- {getUint16,getInt16}/return-values-custom-offset.js
- {getUint32,getInt32}/return-values-custom-offset.js

Could you select byte values that differ between those two views?

**Comment by leobalter** [comment]
> Could you select byte values that differ between those two views?

Great idea. I just did it, thanks!

**Comment by jugglinmike** [comment]
Well in that case, I believe our work here is done.

**Comment by leobalter** [comment]
yay!

**Comment by jugglinmike** [comment]
Merged to `master` at the following commits:

- 9565f01 Add tests for DataView.prototype.getUint8
- 1567b68 Add tests for Dataview.prototype.getInt8
- 5e767dc Add tests for DataView.prototype.getUint16
- 673b63e Add tests for DataView.prototype.getInt16
- 7f8e559 Add tests for DataView.prototype.getUint32
- 0d5b6ea Add tests for DataView.prototype.getInt32
- c1cdf88 Add tests for DataView.prototype.getFloat64
- 47b323c Add tests for DataView.prototype.getFloat32

**Pull request closed without merging**